.github/actions/install-pkg/action.yml

```
@@ -0,0 +1,23 @@
+name: Install earthaccess Python package
+description: Install earthaccess Python package and testing dependencies'
+
+inputs:
+  python-version:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: pip
+
+    - name: Display full python version
+      shell: bash
+      id: full-python-version
+      run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
```

This syntax is deprecated, per https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.

Instead, it should be:

```suggestion
      run: echo "{version}=$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")" >> $GITHUB_OUTPUT
```

---

Nice! Good catch, I just copied it without paying close attention :)